feature/MTM-46143-Add-requirements-for-managing-data (#1043)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/data-retention.md

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

- To view retention rules: READ permission for the permission type "Retention rules"
- To manage retention rules (create, update, delete): ADMIN permission for the permission type "Retention rules"

The above permissions can be used to create roles for robust user management. Every new tenant have specified typical roles by default:
- Tenant Manager - Can manage tenant wide configurations like applications, tenant options and retention rules.

{{< /c8y-admon-req >}}

{{< c8y-admon-related >}}
- [Enterprise tenant > Managing tenants > Tenant policies](/users-guide/enterprise-tenant/#tenant-policies) in the <i>User guide</i> for details on the creation of tenant policies and retention rules on tenant level.
- [Retention rules](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Retention-rules) in the {{< openapi >}} for details on managing retention rules via REST.
{{< /c8y-admon-related >}}

{{< c8y-admon-info >}}

> Retention rules are usually run during the night. When you edit a retention rule you don´t see an immediate effect in the **Usage** section on the Home screen of the Administration application.

Reviewer: "during the night" of the server timezone, for some users it might be different part of the day, right?

Author: What do you think if I change it like?

```markdown
Retention rules are usually run once per calendar day.
```

Author: changed

> Retention rules are usually run during the night. When you edit a retention rule, you will not see an immediate effect in the **Usage** section on the **Home** screen of the Administration application.

Reviewer 2: ```suggestion
Retention rules are usually run during the night. When you edit a retention rule you don´t see an immediate effect in the **Usage** section on the Home screen of the Administration application.
```

Reviewer 2: @Airdroplet Should we perhaps better call it Home dashboard as we do in the Cockpit?

The file repository provides an overview of the files stored in your account.

{{< c8y-admon-req >}} 
ROLES & PERMISSIONS:

> - To view files in the files repository: READ permission for the permission type "Inventory" . You can remove owned files with this permission, but you can not remove files of other users.

Reviewer 2: ```suggestion
- To view files in the files repository: READ permission for the permission type "Inventory" . You can remove owned files with this permission but you cannot remove files of other users.
```

- To upload and manage files of all owners in the files repository: ADMIN permission for the permission type "Inventory".

> - To upload files to the files repository: CREATE permission for the permission type "Inventory".

Reviewer 2: I would change the order of these two permissions as ADMIN is stronger than CREATE.
